src/libs/browser-storage.js

```diff
@@ -32,6 +32,19 @@ export const getStatic = (storage, key) => {
   return maybeParseJSON(storage.getItem(key))
 }
 
+/**
+ * Sets a key/value in browser storage, or removes the key from storage if `value` is undefined.
+ *
+ * Static storage will not be automatically deleted in the case of space overflow. If your storage usage
+ * is transient (for example, navigating between routes and preserving data where the loss of it is not
+ * critical), instead use `setDynamic`.
+ *
+ * See also `staticStorageSlot`, which can be passed to the `useStore` hook function.
+ *
+ * @param storage either `localStorage` or `sessionStorage`
```

Comment:
nit: Could we call this out as something like (I leave it to your discretion if you prefer what you already have): 
```
either one of these `window` properties: `localStorage` or `sessionStorage`
```